Lps 166513 for peer review

hhtank opened the pull request:
@thabet91 When you're comfortable, please help:
- Review this PR, importantly check the correctness of the cleanup logic.
- You can also pull my branch LPS-166513 and verify in local.

Outstanding points that i'm working on are:
- Check if there could be any chance of collision happens in race conditions of ActionableDynamicQuery.
- Check if there is exception thrown when deleting the background task, how would the progress be impacted?

Feel free to let me know or send a PR if you want to suggest any changes. Thanks!

cc @tototrinh I'm open if you can help doing a peer revie on this PR. Thanks!

sergiojimcos committed:
LPS-172037 Add test for UPSERT nested entities

sergiojimcos committed:
LPS-172037 Update addObjectEntry and putObjectentry to upsert nested entities and link them in a relationship

sergiojimcos committed:
LPS-172037 Create new method to parse all nestedObjectEntryProperties into an dto ObjectEntry and include some validations

JanisZhang committed:
LPS-175362 Remove stream usages in GroupSelectorTagTest

JanisZhang committed:
LPS-175362 Remove stream usages in ItemSelectorImplTest

ccorreagg committed:
LPS-172037 Add interface to parse a element (could be one entry or a list of them) to a list of elements based on the ObjectDefinition destination of the relationship and the relationship type

Co-authored-by: Sergio Jiménez del Coso <sergio.jdelcoso@liferay.com>

ccorreagg committed:
LPS-172037 Add implementation of the parsers registry

Co-authored-by: Sergio Jiménez del Coso <sergio.jdelcoso@liferay.com>

ccorreagg committed:
LPS-172037 Register the Parsers for oneToMany and ManyToMany relationship types

Co-authored-by: Sergio Jiménez del Coso <sergio.jdelcoso@liferay.com>

thabet91 committed:
LPS-166513 adding time config for export/import

ces-totrinh committed:
LPS-166513 Initialize implementation of Cleanup expired background tasks of Sites and Portlets

ccorreagg committed:
LPS-172037 Add implementations of the parser for OneToMany and ManyToMany relationships for the ObjectDefinitions that are not system object

Co-authored-by: Sergio Jiménez del Coso <sergio.jdelcoso@liferay.com>

ccorreagg committed:
LPS-172037 Use Parser instead

Co-authored-by: Sergio Jiménez del Coso <sergio.jdelcoso@liferay.com>

ccorreagg committed:
LPS-172037 Move method to an Util

Co-authored-by: Sergio Jiménez del Coso <sergio.jdelcoso@liferay.com>

ccorreagg committed:
LPS-172037 Remove useless validations

veroglez committed:
LPS-168542 Replace remove links by remove buttons and add aria-label and title for each one

veroglez committed:
LPS-168525 Replace select link by select button and add more context in aria-label

veroglez committed:
LPS-168525 Remove headers

Kvale1733 committed:
LRQA-79442 Need to sign in again on WebLogic and Websphere if restarting portal

Kvale1733 committed:
LRQA-79442 Run tests that use mysql macros only on mysql

Kvale1733 committed:
LRQA-79442 Explicitly set test.assert.warning.exceptions to false for test UninstallJarOffline

thabet91 committed:
LPS-166513 updating language keys; SF

Kvale1733 committed:
LRQA-79442 Run test only on tomcat

Kvale1733 committed:
LRQA-79442 Add missing pql so tests don't run on the wrong environment

BarbaraCabrera committed:
LPS-176279 Web Content list UI adjustments

Change the List view for Web Content and Web Content Folder to show the correct order of the information and migrate the aui:a to a clay:link

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sergiojimcos committed:
LPS-172037 Use FeatureFlagManagerUtil instead of PropsUtil.get(FF)

allen-ziegenfus committed:
ISSITES-5990 Do not automatically publish new apps

allen-ziegenfus committed:
ISSITES-5990 SKUs should not expire

javalosjr96 committed:
LPS-176295 Upgrade Azure Core Libraries to 1.2.10 which can resolve one performance issue, please see Azure/azure-sdk-for-java@af97f53.

In addition, 1.0.4‘s dependencies should exist Vulnerabilities issue , please see https://mvnrepository.com/artifact/com.azure/azure-sdk-bom/1.0.4.

javalosjr96 committed:
LPS-176295 Exclude new packages and adjust fixupmessages

javalosjr96 committed:
LPS-176295 SF,Simplify

michaelhashimoto committed:
LRCI-3346 Allow for creating multiple Jenkins WebHook references

michaelhashimoto committed:
LRCI-3346 Add jenkins event handlers for Builds

sergiojimcos committed:
LPS-172037 Delete toObjectEntry from ObjectEntryValuesUtil and implement it on BaseObjectRelationshipElementsParserImpl

sergiojimcos committed:
LPS-172037 baseline

brianchandotcom committed:
LPS-172037 Wordsmith

brianchandotcom committed:
LPS-172037 SF

brianchandotcom committed:
LPS-172037 Rename

brianchandotcom committed:
LPS-172037 auto SF

michaelhashimoto committed:
LRCI-3346 Add Description & Descriptor classes to recall configured data from the database

michaelhashimoto committed:
LRCI-3346 Add jenkins event handlers for Computers

michaelhashimoto committed:
LRCI-3346 Add jenkins event handlers for Queue Items

Kvale1733 committed:
LRQA-79524 Disable ci retry on stable core tests

hhtank committed:
LPS-166513 Realize logic of cleanup expired Export entries

hhtank committed:
LPS-166514 Cleanup Import entries

hhtank committed:
LPS-166513 baseline

hhtank commented:
@thabet91 @tototrinh 
FYI, i've pushed the latest changes which capture the requirement changes via discussion with the PO/PM.
I'm pretty sure this could be the version we're going to send to Staging for a review.
If you have time, please help take another look and give me feedbacks if needed.
@thabet91 Technically you can take this PR and run in your local and can start writing Tech Drafts.

Thanks,

ces-totrinh commented:
Look good to me. Thanks.

hhtank committed:
LPS-166513 Cleanup cancelled entries

Commit:
LPS-176295 Remove unnecessary packages from import-package. I removed all packages from import-package. And then checked generated Import-Package from MANIFEST.MF. Generated Import-Package doesn't import these packages, so I remove them.

danielCamara02 committed:
LPS-176481 Add CanPreviewAfterChangeFragmentBeforePublishing test

sergiojimcos committed:
LPS-172037 Change generic for ObjectEntry

michaelhashimoto committed:
LRCI-3346 Simplify methods to get various jenkins json objects

gamerson committed:
LPS-177370 remove oauth2 sample related code and move to refresh_sample_workspace.sh

gamerson committed:
LPS-177370 update to latest versions

gamerson committed:
LPS-177370 refresh

izaera committed:
LPS-171612 Automate the creation of the angular sample

izaera committed:
LPS-171612 Move client-extension.yaml creation to function

I considered extracting a common function for Angular and React, but the
files are so different that I feel it doesn't justify the complexity it
would introduce.

izaera committed:
LPS-171612 UseESM should be true for every custom element

Otherwise they won't be able to import any DXP API and/or use DXP's React.

izaera committed:
LPS-171612 Refresh sample workspaces

jeffwu0724 committed:
LPS-177368 remove remove stream and optional usage in CommerceShippingFixedOptionLocalServiceImpl. Use a for loop instead of ListUtil.filter here to avoid making a new List. Also break the loop when we find the first match to reduce the number of iterations

Kvale1733 committed:
LRQA-79381 Use bare database for RenameFinderColumn test

Commit:
LRAC-13080 Update CI rules in test.properties file with new branches

michaelhashimoto committed:
LRCI-3346 Rename JenkinsWebHook -> JenkinsPublisher

ealonso committed:
LPS-177536 Delete test modules

ealonso committed:
LPS-177537 Delete web modules without portlet

ealonso committed:
LPS-177537 Removes suppression

ealonso committed:
LPS-177538 Delete modules without data in the DB

brianchandotcom committed:
artifact:ignore list-type-service 1.0.34 change log

brianchandotcom committed:
artifact:ignore list-type-service 1.0.34 prep next

brianchandotcom committed:
artifact:ignore list-type-service 1.0.34 artifact properties

brianchandotcom committed:
artifact:ignore notification-rest-impl 1.0.27 change log

brianchandotcom committed:
artifact:ignore notification-rest-impl 1.0.27 prep next

brianchandotcom committed:
artifact:ignore notification-rest-impl 1.0.27 artifact properties

brianchandotcom committed:
artifact:ignore notification-service 1.0.41 change log

brianchandotcom committed:
artifact:ignore notification-service 1.0.41 artifact properties

brianchandotcom committed:
artifact:ignore notification-service 1.0.41 prep next

brianchandotcom committed:
artifact:ignore notification-web 1.0.39 change log

brianchandotcom committed:
artifact:ignore notification-web 1.0.39 prep next

brianchandotcom committed:
artifact:ignore notification-web 1.0.39 artifact properties

brianchandotcom committed:
artifact:ignore object-admin-rest-api 16.1.0 change log

brianchandotcom committed:
artifact:ignore object-admin-rest-api 16.1.0 prep next

brianchandotcom committed:
artifact:ignore object-admin-rest-api 16.1.0 artifact properties

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.94 change log

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.94 artifact properties

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.94 prep next

brianchandotcom committed:
artifact:ignore object-service 1.0.137 change log

brianchandotcom committed:
artifact:ignore object-service 1.0.137 artifact properties

brianchandotcom committed:
artifact:ignore object-service 1.0.137 prep next

brianchandotcom committed:
artifact:ignore object-storage-salesforce 1.0.21 change log

brianchandotcom committed:
artifact:ignore object-storage-salesforce 1.0.21 prep next

brianchandotcom committed:
artifact:ignore object-storage-salesforce 1.0.21 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.91 change log

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.91 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.91 prep next

brianchandotcom committed:
artifact:ignore headless-admin-list-type-impl 1.0.43 change log

brianchandotcom committed:
artifact:ignore headless-admin-list-type-impl 1.0.43 artifact properties

brianchandotcom committed:
artifact:ignore headless-admin-list-type-impl 1.0.43 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-analytics 5.0.9 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-analytics 5.0.9 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-form-analytics 5.0.9 artifact properties

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.81 change log

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.81 artifact properties

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.81 prep next

brianchandotcom committed:
artifact:ignore object-web 1.0.153 change log

brianchandotcom committed:
artifact:ignore object-web 1.0.153 artifact properties

brianchandotcom committed:
artifact:ignore object-web 1.0.153 prep next

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.79 change log

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.79 prep next

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.79 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-machine-learning-forecast-alert-service 5.0.27 change log

brianchandotcom committed:
artifact:ignore commerce-machine-learning-forecast-alert-service 5.0.27 prep next

brianchandotcom committed:
artifact:ignore commerce-machine-learning-forecast-alert-service 5.0.27 artifact properties

brianchandotcom committed:
artifact:ignore multi-factor-authentication-email-otp-service 2.0.19 change log

brianchandotcom committed:
artifact:ignore multi-factor-authentication-email-otp-service 2.0.19 artifact properties

brianchandotcom committed:
artifact:ignore multi-factor-authentication-email-otp-service 2.0.19 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-fido2-credential-service 4.0.24 change log

brianchandotcom committed:
artifact:ignore multi-factor-authentication-fido2-credential-service 4.0.24 artifact properties

brianchandotcom committed:
artifact:ignore multi-factor-authentication-fido2-credential-service 4.0.24 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-timebased-otp-service 2.0.20 change log

brianchandotcom committed:
artifact:ignore multi-factor-authentication-timebased-otp-service 2.0.20 prep next

brianchandotcom committed:
artifact:ignore multi-factor-authentication-timebased-otp-service 2.0.20 artifact properties

brianchandotcom committed:
artifact:ignore portal-reports-engine-console-service 8.0.35 change log

brianchandotcom committed:
artifact:ignore portal-reports-engine-console-service 8.0.35 prep next

brianchandotcom committed:
artifact:ignore portal-reports-engine-console-service 8.0.35 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.98 change log

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.98 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.98 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-service 5.0.36 change log

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-service 5.0.36 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-forms-service 5.0.36 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-client 3.0.20 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-client 3.0.20 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-client 3.0.20 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.64 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.64 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-rest-impl 3.0.64 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.49 change log

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.49 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-metrics-service 3.0.49 prep next

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.48 change log

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.48 prep next

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.48 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-api 24.0.0 change log

brianchandotcom committed:
artifact:ignore search-experiences-api 24.0.0 prep next

brianchandotcom committed:
artifact:ignore search-experiences-api 24.0.0 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-rest-client 3.0.12 change log

brianchandotcom committed:
artifact:ignore search-experiences-rest-client 3.0.12 prep next

brianchandotcom committed:
artifact:ignore search-experiences-rest-client 3.0.12 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.36 change log

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.36 prep next

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.36 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-rest-client 3.0.16 change log

brianchandotcom committed:
artifact:ignore segments-asah-rest-client 3.0.16 prep next

brianchandotcom committed:
artifact:ignore segments-asah-rest-client 3.0.16 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-rest-impl 3.0.42 change log

brianchandotcom committed:
artifact:ignore segments-asah-rest-impl 3.0.42 prep next

brianchandotcom committed:
artifact:ignore segments-asah-rest-impl 3.0.42 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-service 5.0.25 change log

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-service 5.0.25 prep next

brianchandotcom committed:
artifact:ignore sharepoint-rest-oauth2-service 5.0.25 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-client 1.0.14 change log

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-client 1.0.14 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-client 1.0.14 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.50 change log

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.50 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-machine-learning-impl 5.0.50 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore search-experiences-rest-impl 3.0.31 change log

brianchandotcom committed:
artifact:ignore search-experiences-rest-impl 3.0.31 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-rest-impl 3.0.31 prep next

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.34 change log

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.34 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-service 3.0.34 prep next

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.34 change log

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.34 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.34 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
7.4.3.67 GA67 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream

brianchandotcom committed:
7.4.13 Update 67 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream, antivirus, enterprise-product-notification, multi-factor-authentication, portal-search-elasticsearch-cross-cluster-replication, portal-search-elasticsearch-monitoring, portal-search-learning-to-rank, portal-search-similar-results, portal-search-tuning, saml, search-experiences

JanisZhang committed:
LPS-174127 Remove stream usages in AssetVocabularyConfigurationOptionsProviderTest

JanisZhang committed:
LPS-174127 Remove stream usages in EntityFieldConfigurationFieldOptionsProviderTest

JanisZhang committed:
LPS-174127 SF, generally the looped item goes second.

Hanlf committed:
LPS-176793 Extend the expiration time to ensure that the one drive doc is added successfully

Commit:
LPS-176481 Remove the unnecessary step

austinchiang committed:
LPS-174720 Quarantine test

JanisZhang committed:
LPS-175362 SF, rename variable

austinchiang committed:
LPS-173230 Scroll tags section into view for easier debugging in case of failure

brianchandotcom committed:
Revert "7.4.3.67 GA67 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream"

This reverts commit 791fc5fa6da72b32462e7898ce10fd2030ce4fb5.

brianchandotcom committed:
Revert "7.4.13 Update 67 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream, antivirus, enterprise-product-notification, multi-factor-authentication, portal-search-elasticsearch-cross-cluster-replication, portal-search-elasticsearch-monitoring, portal-search-learning-to-rank, portal-search-similar-results, portal-search-tuning, saml, search-experiences"

This reverts commit d05febc1ea525a9c05d2fdbbbe5d7bd2eb163420.

brianchandotcom committed:
frontend-js-metal-web 5.0.12 packageinfo

brianchandotcom committed:
artifact:ignore frontend-js-metal-web 5.0.12 artifact properties

brianchandotcom committed:
artifact:ignore frontend-js-metal-web 5.0.12 prep next

brianchandotcom committed:
artifact:ignore frontend-js-components-web 2.0.46 change log

brianchandotcom committed:
artifact:ignore frontend-js-components-web 2.0.46 prep next

brianchandotcom committed:
artifact:ignore frontend-js-components-web 2.0.46 artifact properties

brianchandotcom committed:
Upload site initializers

brianchandotcom committed:
artifact:ignore site-initializer-raylife-d2c 1.0.112 artifact properties

brianchandotcom committed:
artifact:ignore site-initializer-raylife-d2c 1.0.112 prep next

brianchandotcom committed:
artifact:ignore site-initializer-raylife-ap 1.0.69 artifact properties

brianchandotcom committed:
artifact:ignore site-initializer-raylife-ap 1.0.69 prep next

brianchandotcom committed:
artifact:ignore site-initializer-testray 1.0.25 prep next

brianchandotcom committed:
artifact:ignore site-initializer-testray 1.0.25 artifact properties

Commit:
LPS-176279 Modify locator to adapt new markup

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-service 4.0.44 change log

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-service 4.0.44 artifact properties

brianchandotcom committed:
artifact:ignore commerce-shipping-engine-fixed-service 4.0.44 prep next

brianchandotcom committed:
artifact:ignore portal-store-azure 5.0.5 change log

brianchandotcom committed:
artifact:ignore portal-store-azure 5.0.5 prep next

brianchandotcom committed:
artifact:ignore portal-store-azure 5.0.5 artifact properties

brianchandotcom committed:
Rerun lib/versions

brianchandotcom committed:
7.4.3.67 GA67 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream

brianchandotcom committed:
7.4.13 Update 67 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, feature-flag, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, ip-geocoder, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, on-demand-admin, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream, antivirus, enterprise-product-notification, multi-factor-authentication, portal-search-elasticsearch-cross-cluster-replication, portal-search-elasticsearch-monitoring, portal-search-learning-to-rank, portal-search-similar-results, portal-search-tuning, saml, search-experiences

Commit:
LPS-176481 Add macro to view fragment preview page

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-177567 Comment until FF removed

Commit:
LPS-176481 Add steps to view fragment preview page

brianchandotcom committed:
LRCI-3346 Rename

brianchandotcom committed:
LRCI-3346 Is this possible or was the line break and order on purpose? If so, revert this.

brianchandotcom committed:
LRCI-3346 SF

brianchandotcom committed:
LRCI-3346 Chain some #sfme

brianchandotcom committed:
LRCI-3346 Sort

brianchandotcom committed:
LRCI-3346 Sort

brianchandotcom committed:
LRCI-3346 Sort

Jeremy-Ch committed:
LPS-177549 Assert the not found page

Jeremy-Ch committed:
LPS-177551 Update the URL for the onedrive

BarbaraCabrera committed:
LPS-177218 Remove the use of the control-menu-nav-item-content class to display the number of items properly

dantewang committed:
LPS-175362 SF

brianchandotcom committed:
LPS-172037 Rename

brianchandotcom committed:
LPS-172037 SF

brianchandotcom committed:
LPS-172037 Use Collections

brianchandotcom committed:
LPS-172037 Never do it in the base because you have to deal with parseOne scenario which requires validation

Commit:
LPS-177550 Modify cases to make them reusable

brianchandotcom committed:
LPS-172037 a ClassCastException should eventually result in a BadRequest (if not, please revert this or add some kind of exception mapper for bad input)

brianchandotcom committed:
LPS-172037 Rename

brianchandotcom committed:
LPS-172037 auto SF

Jeremy-Ch committed:
LPS-177548 Assert the correct selection message

brianchandotcom committed:
LPS-168542 SF

brianchandotcom committed:
LPS-171612 Sort

brianchandotcom committed:
LPS-171612 Rename

brianchandotcom committed:
LPS-171612 Type yes

brianchandotcom committed:
LPS-171612 Make it local

brianchandotcom committed:
LPS-171612 Wordsmith

brianchandotcom committed:
LPS-171612 regen

brianchandotcom committed:
LPS-171612 Don't do anything

brianchandotcom committed:
LPS-171612 Rename create_remote_app.sh to create_custom_element.sh

brianchandotcom committed:
LPS-171612 Title or sentence

brianchandotcom committed:
LPS-177370 refresh other

hhtank committed:
LPS-166513 Adjust message keys

hhtank committed:
LPS-166513 Add language messages

hhtank committed:
LPS-166513 buildLang
